### ISSUE TITLE
Keyword search on river throws error

### ISSUE THREAD

**Issue by dan-king** [ushahidi/SwiftRiver-API#37]
Using the keyword parameter throws an unknown error.

This example returns all drops for River #31 as expected:

```
    /rivers/31/drops
```

However, adding keyword search throws an error.

```
   /rivers/31/drops?keywords=test
```

"Fatal error: Uncaught exception 'Exception' with message 'An error occurred: array ( 'message' => 'Unknown error', )"

Note, the documentation for the keywords request is incomplete. The example request is simply "GET" with no parameters listed.

(See https://wiki.ushahidi.com/display/WIKI/SwiftRiver+API+River+Resources#SwiftRiverAPIRiverResources-GETrivers/:id/drops and look for the example "The below request will return 1 Facebook OR Twitter drops with an id less than 999999 AND containing the keywords Kenya OR Uganda AND have a location of San Francisco OR New York City.")

**Comment by ekala** [comment]
@dan-king I suspect the default search field - `df` config parameter -  for the `/select` request handler in your `solrconfig.xml` hasn't been defined/set. 

_NOTE_: Whenever the `keywords` parameter is specified in a `GET rivers/:id/drops` or `GET buckets/:id/drops`, we delegate the search for the specified keyword(s) to Solr. Therefore, if Solr happens to miss some mandatory configuration, the API returns an error. I'm in the process of documenting all this on the wiki.

I've rectified the documentation for `GET rivers/:id/drops` on the wiki; I think it may have broken when we upgraded the wiki software a couple of weeks ago. Thanks for the note!
